examples/hooks-remix/app/routes/index.tsx

```diff
@@ -0,0 +1,122 @@
+import algoliasearch from 'algoliasearch/lite';
+import type { InstantSearchServerState } from 'react-instantsearch-hooks-web';
+import {
+  DynamicWidgets,
+  Hits,
+  InstantSearch,
+  InstantSearchSSRProvider,
+  Pagination,
+  RefinementList,
+  SearchBox,
+  useInstantSearch,
+} from 'react-instantsearch-hooks-web';
+import { getServerState } from 'react-instantsearch-hooks-server';
+import { history } from 'instantsearch.js/cjs/lib/routers/index.js';
+import instantSearchStyles from 'instantsearch.css/themes/satellite-min.css';
+
+import type { LinksFunction, LoaderFunction } from '@remix-run/node';
+import { json } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+
+import { Hit } from 'components/Hit';
+import { Panel } from 'components/Panel';
+import { ScrollTo } from 'components/ScrollTo';
+
+import tailwindStyles from '../tailwind.css';
```

Comment:
I don't see the file in this project.

Reply:
This file gets generated when running [`yarn dev`](https://github.com/algolia/react-instantsearch/pull/3575/files#diff-cce33214553e43b59c0f7a4fb28cf48a4040ba20d105e608cc700d4c4fd7a911R10). That's [how it's recommended on the Remix docs](https://remix.run/docs/en/v1/guides/styling#tailwind-css).